linux_sql/scripts/sql/ddl.sql

```diff
@@ -0,0 +1,60 @@
+postgres=# \c host_agent;
```

Comment:
It's a good idea to drop any prexisting db and create it before connecting. One less command to run, and dropping ensures you aren't colliding with previous attempts.